chapter2/scenarios/06_Dwarven_Doors.cfg

```diff
@@ -513,15 +513,15 @@
     {NOTRAIT_UNIT 2 Swordsman 7 7}
     [message]
       type=Swordsman
-      message=_"Are you finished with the armors?"
+      message=_"Are you finished with the armour suits?"
     [/message]
     [message]
       type=Dwarvish Runesmith
       message=_"Yes, I am."
     [/message]
     [message]
       type=Swordsman
-      message=_"Ok let me see. (examining armors) They are very strong and nice."
+      message=_"Ok let me see. (examining armour) They are very strong and nice."
```

Comment:
My understanding is that the plural of armour / armor is still armour / armor. Armors isn't a word according to the dictionary I use.  If you look at my changes I did put in "armour suits" so the suits makes it plural. 

I'm from Australia so follow the British English spelling generally, but totally understand that the main audience is probably American. 




On 14 June 2016 3:28:59 am AEST, "Jakub Konečný" <notifications@github.com> wrote:
>>      [/message]
>>      [message]
>>        type=Dwarvish Runesmith
>>        message=_"Yes, I am."
>>      [/message]
>>      [message]
>>        type=Swordsman
>> -      message=_"Ok let me see. (examining armors) They are very
>strong and nice."
>> +      message=_"Ok let me see. (examining armour) They are very
>strong and nice."
>
>The idea is that he made several suits, not just one. Oh, and I  think
>that armor is correct in American(?) English.
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/konecnyjakub/For_Power/pull/13/files/e215a2eb16e60c9e42b3a8dfc2ee977fe24805da..4093722429f8e45b43f4b1bffd0d4198f8544927#r66832786

Regards, 
Rod

Automation Concepts 
p: 0416 702 742 
e: rod@automationconcepts.com.au 
a: PO Box 384 North Lakes Qld 4509